openvpn/version.go

```diff
@@ -0,0 +1,38 @@
+package openvpn
+
+import (
+	"os/exec"
+	"syscall"
+)
+
+//Version function checks that openvpn is available, given path to openvpn binary
+func Version(openvpnBinary string) error {
```

Comment:
Function name declares implementation, not the purpose of this function. `CheckOpenVPNBinary` ?

Reply:
The idea was to make a function whchi returns (string, error) with string as openvpn version and error if exec somehow fails. But for the moment yes - it can be named CheckOpenVpnBinary